src/pytest_mysql/executor.py

```diff
@@ -77,7 +77,7 @@ def version(self):
         version_output = subprocess.check_output(
             [self.mysqld, '--version']
         ).decode('utf-8')
-        return self.VERSION_RE.match(version_output).groupdict()['version']
+        return self.VERSION_RE.search(version_output).groupdict()['version']
```

Comment:
funny issue is that without these changes, the tests passes as well on my machine. Is it possible that problem is somewhere else?

Reply:
the only case where it helps, is when the output is two lines actually (first being empty)

Reply:
@fizyk Weird. It wouldn't work without these changes on my Mac at work, and neither on my co-workers.

Reply:
If you could try to check what's the exact `version_output` value that would be great.

Reply:
@fizyk the exact values were described in the issue #107 :) I am surprised how this works differently on different systems.